include/range/v3/view/any_view.hpp

```diff
@@ -443,17 +541,24 @@ namespace ranges
                 ptr_ = (that.ptr_ ? that.ptr_->clone() : nullptr);
                 return *this;
             }
+
+            CONCEPT_REQUIRES((Cat & category::sized) == category::sized)
```

Comment:
the member function still needs to be here (rather than in ` any_view_size_base<true>`) to override the inherited interface's virtual size() member.

Reply:
Is this comment referring to the `size()` member function? It can't be overriding anything because it's a template, so I don't understand your comment.

Reply:
The comment was based on the previous implementation with `any_view_size_base` (and wrong assumptions by myself).